internal/util/get/get_test.go

```diff
@@ -94,14 +96,16 @@ func TestCommand_Run_subdir(t *testing.T) {
 	g, w, clean := testutil.SetupDefaultRepoAndWorkspace(t, testutil.Dataset1)
 	defer clean()
 
+	defer testutil.Chdir(t, w.WorkspaceDirectory)()
+
+	r := filepath.Join(w.WorkspaceDirectory, subdir)
```

Comment:
Nit: Rename variable to absPath instead of r ?

Reply:
Updated.